crowdsec-docs/unversioned/console/remediation_metrics.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: remediation_metrics
+title: Remediation Metrics
+---
+
+:::info
+To receive remediation metrics, make sure you’re using **CrowdSec v1.6.3** or higher.  
+**Note:** Not all Remediation Components report metrics to the CrowdSec Console. For details, refer to the [official documentation](https://docs.crowdsec.net/u/bouncers/intro).
+:::
+
+## Introduction
+
+The **Remediation Metrics** page offers a clear and comprehensive view of the malicious activity that CrowdSec has detected and blocked on your infrastructure.  
```

Comment:
We need to stay away from using "blocked" as it can be alot of other remediation types so either use "remediated" or "dropped" as this is the key word on the page.